R/demo-education.R

```diff
@@ -1,20 +1,32 @@
 #' Access to Education Data
 #'
-#' This function returns a data frame from Education in State Cancer Profiles
+#' This function returns a data frame about education demographics from State Cancer Profiles.
 #'
 #' @param area A state/territory abbreviation or USA.
-#' @param areatype Either "county", "hsa" (Health service area), or "state"
-#' @param education Either "less than 9th grade", "at least high school", "at least bachelors degree"
-#' @param sex Either "both sexes", "male", "female"
-#' @param race One of the following values: "All Races (includes Hispanic)", "white (includes hispanic)",
-#'              "white non-hispanic","black","amer. indian/alaskan native (includes hispanic)",
-#'              "asian or pacific islander (includes hispanic)","hispanic (any race)
+#' @param areatype Either `"county"`, `"hsa"` (Health Service Area), or
+#' `"state"`.
+#' @param education The only permissible values are 
+#' - `"less than 9th grade"`
+#' - `"at least high school"`
+#' - `"at least bachelors degree"`
+#' @param sex One of the following values:
+#' - "both sexes"
+#' - "male"
+#' - "female"
+#' @param race One of the following values:
+#' - "All Races (includes Hispanic)"
+#' - "White (includes Hispanic)"
+#' - "White non-Hispanic"
+#' - "Black"
+#' - "Amer. Indian/Alaskan Native (includes Hispanic)"
+#' - "Asian or Pacific Islander (includes Hispanic)"
+#' - "hispanic (Any Race)
 #'
 #' @importFrom httr2 req_url_query req_perform
 #' @importFrom cli cli_abort
 #' @importFrom stats setNames
 #'
-#' @returns A data frame with the following columns: Area Type, Area Code, "Percent", "Households", "Rank"
+#' @returns A data frame with the following columns: Area Type, Area Code, Percent, Households, Rank
```

Comment:
These `@returns` sentences should be complete(ish) sentences, meaning they should end with a period.